docs/7.x/changelog.md

```diff
@@ -33,22 +33,70 @@ These versions are past their End of Support date, and no longer receive securit
+| [6.2](#62-releases) | 6.2.5         | No  | September 24, 2019   | December 18, 2019 (6.3) | 1.16.3               | 3.2.13           |
+| [6.0](#60-releases) | 6.0.10        | No  | July 17, 2019        | August 2, 2019 (6.1)    | 1.14.7               | 3.2.12           |
+| [5.6](#56-releases) | 5.6.8         | No  | April 19, 2019       | July 17, 2019 (6.0)     | 1.14.7               | 3.0.6-gravity    |
+| [5.4](#54-releases) | 5.4.10        | No  | December 14, 2018    | March 8, 2019 (5.5)     | 1.13.5               | 2.4.10           |
+| [5.3](#53-releases) | 5.3.9         | No  | October 19, 2018     | December 14, 2018 (5.4) | 1.12.3               | 2.4.7            |
+| [5.2](#52-releases) | 5.2.18        | Yes | October 15, 2018     | October 15, 2019        | 1.11.9               | 2.4.10           |
+| [5.0](#50-releases) | 5.0.36        | Yes | April 18, 2018       | April 13, 2019          | 1.9.13-gravitational | 2.4.10           |
+| [4.x](#4x-releases) | 4.68.0        | Yes | June 1, 2017         | November 16, 2018       | 1.7.18-gravitational | 2.3.5            |
+| [3.x](#3x-releases) | 3.64.0        | Yes | February 16, 2017    | June 2, 2018            | 1.5.7                | 2.0.6            |
+| [1.x](#1x-releases) | 1.29.0        | Yes | November 2nd, 2016   | March 21, 2018          | 1.3.8                | 1.2.0            |
 
 # Release Notes
 
+## 7.1 Releases
+
+7.1 is currently pre-release.
+
+### 7.1.0-alpha.5 (March 16, 2021)
+
+All changes listed are in comparison to 7.0.30 LTS.
+
+#### Improvements
+* Open source all Gravity code previously limited to enterprise customers! ([#2375](https://github.com/gravitational/gravity/pull/2375))
+* Add support for kubernetes 1.19 ([#2425](https://github.com/gravitational/gravity/pull/2425), [planet#825](https://github.com/gravitational/planet/pull/825), [storage-app#5](https://github.com/gravitational/storage-app/pull/5), [logging-app#76](https://github.com/gravitational/logging-app/pull/76), [monitoring-app#206](https://github.com/gravitational/monitoring-app/pull/206), [bandwagon#33](https://github.com/gravitational/bandwagon/pull/33))
+* Add an Ingress App to gravity ([#1435](https://github.com/gravitational/gravity/pull/1435))
+* Allow OpsCenter and the Web Installation wizard to be disabled in the cluster manifest ([#2371](https://github.com/gravitational/gravity/pull/2371))
+* Allow Logging, Monitoring, Ingress, Tiller, Storage, and Bandwagon applications to be disabled in the cluster manifest ([#2397](https://github.com/gravitational/gravity/pull/2397))
+* Expose additional logrange fields in logging-app ([#2369](https://github.com/gravitational/gravity/pull/2369), [logging-app#75](https://github.com/gravitational/logging-app/pull/75))
+* Allow Kubernetes High Availability mode to be enabled in the cluster manifest ([#2404](https://github.com/gravitational/gravity/pull/2404), [planet#821](https://github.com/gravitational/planet/pull/821))
```

Comment:
This isn't really meant to be an advertised feature, more of a use at your own risk type of thing.

Reply:
I moved this down to the internal changes section and rephrased to

> Experimentally allow Kubernetes High Availability mode to be enabled in the cluster manifest

If you'd like different wording, let me know.